### PR TITLE
Add 'City guide' android demo application for configuration system API

### PR DESCRIPTION
**Pull request by ikulikov**

### CODE REVIEW

File: examples/cityguide/source/src/org/kaaproject/kaa/demo/cityguide/CityGuideActivity.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	@Override
+	public boolean onCreateOptionsMenu(Menu menu) {
+		getMenuInflater().inflate(R.menu.city_guide, menu);
+		return true;
+	}
+
+	@Override
+	public boolean onOptionsItemSelected(MenuItem item) {
+		int id = item.getItemId();
+    	if (id == android.R.id.home) {
```
Comment (ashvayka) [inline examples/cityguide/source/src/org/kaaproject/kaa/demo/cityguide/CityGuideActivity.java]: code style

File: examples/cityguide/source/src/org/kaaproject/kaa/demo/cityguide/adapter/AreasAdapter.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	private Context mContext;
+	private List<Area> mAreas;
+
+	public AreasAdapter(Context context, List<Area> areas) {
+		mContext = context;
+		mAreas = areas;
+	}
+
+    @Override
+    public int getCount() {
```
Comment (ashvayka) [inline examples/cityguide/source/src/org/kaaproject/kaa/demo/cityguide/adapter/AreasAdapter.java:43]: code style

**Comment by ashvayka** [comment]
Great example, however need to apply code formatting for all project

File: examples/cityguide/source/src/org/kaaproject/kaa/demo/cityguide/util/Utils.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+		final int buffer_size = 1024;
+		try {
+			byte[] bytes = new byte[buffer_size];
+			for (;;) {
+				int count = is.read(bytes, 0, buffer_size);
+				if (count == -1)
+					break;
+				os.write(bytes, 0, count);
+			}
+		} catch (Exception ex) {
```
Comment (shpigor) [inline examples/cityguide/source/src/org/kaaproject/kaa/demo/cityguide/util/Utils.java]: At least need to log this exception

**Comment by shpigor** [comment]
Reviewed

**Pull request merged**